### ISSUE TITLE
Skript Randomly Forgets New 1.21 Items

### ISSUE THREAD

**Issue by aaaeeeMC** [SkriptLang/Skript#6946]
### Skript/Server Version

```markdown
[17:17:14 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[17:17:14 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[17:17:14 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[17:17:14 INFO]: [Skript] Server Version: 1.21-109-5a5035b (MC: 1.21)
[17:17:14 INFO]: [Skript] Skript Version: 2.9.0 (skriptlang-github)
[17:17:14 INFO]: [Skript] Installed Skript Addons:
[17:17:14 INFO]: [Skript]  - SkBee v3.5.7 (https://github.com/ShaneBeee/SkBee)
[17:17:14 INFO]: [Skript] Installed dependencies: None
```

### Bug Description

I was coding, like everybody else, then suddenly, when I tried reloading the script, it couldn't recognize the ominous trial key. This was completely random. It doesn't work with any 1.21 items either.

### Expected Behavior

It was obviously supposed to know what an ominous trial key is...

### Steps to Reproduce

I don't know because this was completely random, I'm not sure what caused it. The amount of lines in my script were 615.

### Errors or Screenshots

![image](https://github.com/user-attachments/assets/4ba623aa-b6b2-435b-9fc6-ee8e9cf9105e)
![image](https://github.com/user-attachments/assets/52ad28f1-e8c3-40c3-92e1-74b43585ce1e)

### Other

I'm making a server for a friend and he expects me to finish tomorrow, I'm in huge trouble.

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by Efnilite** [comment]
did you restart

**Comment by APickledWalrus** [comment]
I'm not immediately able to replicate this issue. Are you having any errors in your console?

**Comment by aaaeeeMC** [comment]
> I'm not immediately able to replicate this issue. Are you having any errors in your console?

no

**Comment by aaaeeeMC** [comment]
> did you restart

now skript recognizes the items after the restart. but it still happened.

**Comment by APickledWalrus** [comment]
Are you doing `/sk reload aliases` or `/sk reload all`? I believe that may be causing the issue.

**Comment by aaaeeeMC** [comment]
> Are you doing `/sk reload aliases` or `/sk reload all`? I believe that may be causing the issue.

i didn't reload all, but i did reload aliases but it wasn't right before the issue, it was earlier than that. the issue did not appear right after reloading the aliases, it appeared like 10+ minutes afterwards with several script reloads inbetween.

**Comment by aaaeeeMC** [comment]
> > Are you doing `/sk reload aliases` or `/sk reload all`? I believe that may be causing the issue.
> 
> i didn't reload all, but i did reload aliases but it wasn't right before the issue, it was earlier than that. the issue did not appear right after reloading the aliases, it appeared like an hour afterwards (it also timed me out and sent errors in console) with several script reloads inbetween.

**Comment by APickledWalrus** [comment]
From my testing, this appears to be caused by reloading the aliases. We will have it fixed for 2.9.1 (which releases tomorrow).

**Comment by aaaeeeMC** [comment]
> From my testing, this appears to be caused by reloading the aliases. We will have it fixed for 2.9.1 (which releases tomorrow).

oh, what a delight!
